sdk/servicebus/azure-messaging-servicebus/src/samples/java/com/azure/messaging/servicebus/ServiceBusClientBuilderJavaDocCodeSamples.java

```diff
@@ -44,6 +33,32 @@ public void instantiateReceiverAsync() {
         receiver.receiveMessages().blockFirst(Duration.ofSeconds(1));
     }
 
+    public void instantiateSessionReceiver() {
+        // BEGIN: com.azure.messaging.servicebus.session.receiver.async.client.instantiation
+        // Retrieve 'connectionString', 'topicName' and 'subscriptionName' from your configuration.
+        ServiceBusClientBuilder builder = new ServiceBusClientBuilder()
+            .connectionString(connectionString);
+        // Create a session receiver.
+        ServiceBusSessionReceiverAsyncClient sessionReceiver = builder
+            .sessionReceiver()
+            .receiveMode(ServiceBusReceiveMode.PEEK_LOCK)
+            .topicName(topicName)
+            .subscriptionName(subscriptionName)
+            .buildAsyncClient();
+
+        // Receiving messages from the first available sessions. It waits up to the AmqpRetryOptions.getTryTimeout().
+        // If no session is available within that operation timeout, it completes with an error. Otherwise, a receiver
+        // is returned when a lock on the session is acquired.
+        Mono<ServiceBusReceiverAsyncClient> receiverMono = sessionReceiver.acceptNextSession();
+
+        Disposable subscription = Flux.usingWhen(receiverMono,
```

Comment:
Why do we assign this to a subscription? ` subscription.dispose();` is not part of the snippet so, the assignment may not be required.